### ISSUE TITLE
Issues with update 1.8.0 (Fallthrough Attributes)

### ISSUE THREAD

**Issue by N1cE-me** [nuxt/icon#300]
After update with `Suspense` I can't pass class & event listeners for NuxtIcon component.

For example:
```vue
<Icon name="mdi:puzzle" class="absolute h-[40px] w-6 cursor-pointer rounded text-emerald-600" />
```
  
Now rendered as:

```html
<span class="iconify i-mdi:puzzle" aria-hidden="true"></span>
```

Instead correct v1.7.6
```html
<span class="iconify i-mdi:puzzle absolute h-[40px] w-6 cursor-pointer rounded text-emerald-600" aria-hidden="true"></span>
```

And with warnings:
```console
 WARN  [Vue warn]: Extraneous non-props attributes (class) were passed to component but could not be automatically inherited because component renders fragment or text or teleport root nodes.
  at <NuxtIcon name="mdi:puzzle" class="h-[40px] w-6 cursor-pointer rounded text-emerald-600" >
  at <RouterLink ref=undefined to="/" activeClass=undefined  ... >
  at <NuxtLink href="/" >
  at <Error error= options.customise is not a function

  <span class="stack internal">at getCSS (node_modules\@nuxt\icon\dist\runtime\components\css.js:93:41)</span>
  <span class="stack internal">at node_modules\@nuxt\icon\dist\runtime\components\css.js:169:25</span></pre> >
  at <NuxtRoot>
```

Same issue with event listeners, for example:
```vue
<Icon name="ant-design:plus-outlined" class="icon" @click="popoverOpenIndex = index" @dblclick.stop />
```

`@click` not work now

With `<Icon :suspensible="false" ... />`  same issue

----
`vue`: 3.5.13
`nuxt`: 3.14.159
`@nuxt/icon`: 1.8.0
node: 22.10.0

**Comment by wangziling** [comment]
The same issue. 🤔

**Comment by N1cE-me** [comment]
fixed with v1.8.1: 291f5118c122f6a24d2caf11092865ce9b020def

Thank you!
